windows/RNTrackPlayer.sln

```diff
@@ -1,89 +1,289 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.16
+# Visual Studio Version 16
```

Comment:
I'm frankly not sure about all these changes in the .sln file. It'd be good for someone that knows what they're doing to look at it.